tests/unit/v1/test_watch.py

```diff
@@ -278,7 +292,7 @@ def test_for_query(self):
         parent = DummyCollection(client)
         modulename = "google.cloud.firestore_v1.watch"
         pb2 = DummyPb2()
-        with mock.patch("%s.firestore_pb2" % modulename, pb2):
+        with mock.patch("%s.firestore" % modulename, pb2):
```

Comment:
Should this be updated to use f-strings?

Reply:
Resolving as we have an open issue: https://github.com/googleapis/python-firestore/issues/94